converter/internal/promtailconvert/testdata/windowsevents_relabel.river

```diff
@@ -18,6 +18,7 @@ loki.source.windowsevent "fun" {
 	exclude_user_data      = true
 	use_incoming_timestamp = true
 	forward_to             = [loki.relabel.fun.receiver]
+	labels                 = {}
```

Comment:
It would yeah. But this function is used in many other places and I see that loki write has the same problem with the external labels using a duplicate of this function. I could move the function into common and have it return nil instead of empty map but I would do it in another PR since that it is affecting features beyond the scope of this PR

Reply:
I see, if that touches on a lot of places, I think we can skip it for now.